python/python/glide/async_commands/core.py

```diff
@@ -2146,6 +2147,46 @@ async def zrange_withscores(
             Mapping[str, float], await self._execute_command(RequestType.Zrange, args)
         )
 
+    async def zrangestore(
+        self,
+        destination: str,
+        source: str,
+        range_query: Union[RangeByIndex, RangeByLex, RangeByScore],
+        reverse: bool = False,
+    ) -> int:
+        """
+        Stores a specified range of elements from the sorted set at `source`, into a new sorted set at `destination`. If
+        `destination` doesn't exist, a new sorted set is created; if it exists, it's overwritten.
+
+        When in Cluster mode, all keys must map to the same hash slot.
```

Comment:
Should we add `Note: `?
```suggestion
        When in Cluster mode, all `keys` must map to the same hash slot.
```

Reply:
Not sure, I don't mind adding it but the existing docs do not have it and previous comments from Sho on other PRs suggest she prefers following existing doc patterns. Same thing with the angle quotes, keys is not surrounded with angle quotes in the other docs and Sho has commented before asking us to remove them based on this, for example see:
https://github.com/aws/glide-for-redis/pull/1315#discussion_r1573656118
https://github.com/aws/glide-for-redis/pull/1357#discussion_r1582045764

Reply:
We have it for 2 notes, but don't have it for one. Is it ok?

Reply:
Yeah not sure what the AWS preference would be. They don't have a `Note:` before the hash slot warning in the existing docs. But they also don't have any existing docs with both the hash slot warning and the notice about blocking commands.